### PR TITLE
Added 21 systems + 2 custom views to Colorful

### PR DESCRIPTION
**Pull request by Lendersmark**
Added the following systems in Colorful:

Entek Adventure Vision
APF Microcomputer System
Arcadia 2001
Acorn Archimedes
Bally Astrocade
BBC Micro
Camputers Lynx
Commodore Vic 20
Commodore Plus 4

Dragon32
Future Pinball
Nintendo Famicom
Nintendo Super Famicom
Pico-8
Sam Coupe
Sega Model 2
Sega Model 3
SNES MSU-1
Solarus
Sufami Turbo
Sinclair ZX-81
Teknoparrot
Ti-99
Triforce

Also added two custom views, tweaking a bit the colors and the font size:
- Grid + Details (alt)
- Grid + Details + Marquee (alt)

### CODE REVIEW

**Comment by QuaiGoner** [comment]
Is there a way to add Dendy in place of nes, and Sega mega drive 2 ? (

**Comment by CkauNui** [comment]
**Lendersmark**, send me psd for new systems pls (in discord or gmail)

**Comment by Lendersmark** [comment]
> **Lendersmark**, send me psd for new systems pls (in discord or gmail or google drive)
> thx

Sure, could you please give me your email address? or discord channel?

**Comment by CkauNui** [comment]
> > **Lendersmark**, send me psd for new systems pls (in discord or gmail or google drive)
> > thx
> 
> Sure, could you please give me your email address? or discord channel?

ckau48@gmail.com

**Comment by mathsensey** [comment]
Hello,
Is it possible to choose the video (s) that appear randomly for each system?

For example, when I'm on the SNES system, I would like only the Super Mario World video to show.

thank you in advance

**Comment by CkauNui** [comment]
> Hello,
> Is it possible to choose the video (s) that appear randomly for each system?
> 
> For example, when I'm on the SNES system, I would like only the Super Mario World video to show.
> 
> thank you in advance

Yes, put ur mario video in "_inc\anim\video" and name "snes.mp4"

**Comment by mathsensey** [comment]
And it will appear in the TV on the SNES system view? Ok great, I will test this

**Comment by g-spawnPL** [comment]
Hello
I am sending my translation and descriptions in Polish along with graphics for batocera v33 and some of v34 systems. some of them may need improvement because I am not the best in making graphics
[layouts.zip](https://github.com/CkauNui/ckau-book/files/8402478/layouts.zip)
[logos.zip](https://github.com/CkauNui/ckau-book/files/8402479/logos.zip)
adam
advision
amigacdtv
apfm1000
astrocade
bbcmicro
cave3rd
cdi
coco
cplus4
creativision
fm7
gaelco
gamate
gamecom
gamepock
gmaster
hikaru
megaduck
n64dd
naomi2
pc88
pc98
pet
plugnplay
pv1000
samcoupe
supervision
thomson
ti99
tutor
arcadia
apfm1000
cplus4

**Comment by CkauNui** [comment]
**g-spawnPL**, very nice, if u can please send PSD too or make it for dark theme

**Comment by Lendersmark** [comment]
@g-spawnPL wow this is awesome!

**Comment by mathsensey** [comment]
hello, will we have the chance to see Mugen one day? your work is amazing

**Comment by CkauNui** [comment]
**g-spawnPL**, and please sent image in zip, current i no see name in image (161383298-1f877373-7ee4-45d2-ab31-75b920993839.png etc)

**Comment by g-spawnPL** [comment]
> @g-spawnPL wow this is awesome!

thank you but this template is awsome not me :)

**Comment by g-spawnPL** [comment]
> **g-spawnPL**, and please sent image in zip, current i no see name in image (161383298-1f877373-7ee4-45d2-ab31-75b920993839.png etc)

well I just tried to add it in the zip but github won't accept it, you have the names in the zipped layouts - both files must have the same one :)

**Comment by g-spawnPL** [comment]
> **g-spawnPL**, very nice, if u can please send PSD too or make it for dark theme
[gimp_xcf.zip](https://github.com/CkauNui/ckau-book/files/8403919/gimp_xcf.zip)

I was honestly doing for my own purposes and didn't save everything. I use GIMP to edit the attached templates that I found on my PC

**Comment by g-spawnPL** [comment]
if you like it, I'll share the rest of my graphics to this brilliant template when I'm done

I found some more:
[new.zip](https://github.com/CkauNui/ckau-book/files/8403931/new.zip)

**Comment by mathsensey** [comment]
It's possible to add the MUGEN system ? 
Ckau Book is my favorite theme

**Comment by CkauNui** [comment]
**mathsensey**, later

**Comment by CkauNui** [comment]
**g-spawnPL**, mb try use photoshop for best transpency. And i can send psd template for u,
image
image

**Comment by g-spawnPL** [comment]
> **g-spawnPL**, mb try use photoshop for best transpency. And i can send psd template for u, image image

as I wrote I don't know how to create graphics and that's why sometimes it doesn't look the best :) maybe someone from the community could take care of it

**Comment by g-spawnPL** [comment]
> It's possible to add the MUGEN system ? Ckau Book is my favorite theme

I created today for MUGEN and sonicretro, maybe someone will find it usefu
[mugen.zip](https://github.com/CkauNui/ckau-book/files/8404133/mugen.zip)
[sonicretro.zip](https://github.com/CkauNui/ckau-book/files/8404134/sonicretro.zip)
l

**Comment by g-spawnPL** [comment]
Hi
I am uploading the next systems for Batocera V34 with Polish and English translations :-)

[xegs.zip](https://github.com/CkauNui/ckau-book/files/8455706/xegs.zip)
[camplynx.zip](https://github.com/CkauNui/ckau-book/files/8455707/camplynx.zip)
[electron.zip](https://github.com/CkauNui/ckau-book/files/8455708/electron.zip)
[socrates.zip](https://github.com/CkauNui/ckau-book/files/8455709/socrates.zip)
[supracan.zip](https://github.com/CkauNui/ckau-book/files/8455710/supracan.zip)
[vsmile.zip](https://github.com/CkauNui/ckau-book/files/8455711/vsmile.zip)

preview:
xegs
camplynx
electron
socrates
supracan
vsmile

**Comment by g-spawnPL** [comment]
Hi
... next systems for Batocera V34 with Polish and English translations :-)

[archimedes.zip](https://github.com/CkauNui/ckau-book/files/8457538/archimedes.zip)
[atom.zip](https://github.com/CkauNui/ckau-book/files/8457539/atom.zip)

preview:
archimedes
atom

**Comment by Lendersmark** [comment]
Great work @g-spawnPL ! I've updated by fork with your contributions (branch named "colorful-add-systems").

**Comment by g-spawnPL** [comment]
another missing system to the collection :-)

[msxturbor.zip](https://github.com/CkauNui/ckau-book/files/8495701/msxturbor.zip)

preview:
msxturbor

File: _inc/layouts/Spectravideo.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		</text>
+		<text name="TextDesc" extra="true">
+			<size>0.5 0</size>
+			<pos>0.49 0.40</pos>
+			<text>The SV-318 is the basic model of the Spectravideo range. It was fitted with a chiclet style keyboard, difficult to use, alongside which sat a combination cursor pad/joystick. This is a disc-shaped affair with a hole in the centre; put a red plastic 'stick' in the hole and it is a built-in joystick, remove the stick and it is a directional arrow pad for word processing etc. This machine also had only 16 KB of user RAM (plus an additional 16 KB of video RAM), which limited its usefulness, though this could be expanded via an external peripheral box.
+
+This machine is basically identical to its big brother the SV-328, the only differences being in the keyboard and amount of memory. The ROM, expandability, mainboard and case of the two machines were identical.
+
+Reference to the operating system Microsoft Extended BASIC is not to be confused with MSX BASIC, although some marketing at the time claimed that Microsoft Extended is what MSX stood for. The SV-318 is not fully compliant with the MSX standard.</text>
+			<text lang="pl">Descpriptions. Pls send me it in Discord</text>
```
Comment (g-spawnPL) [inline _inc/layouts/Spectravideo.xml:66]: pls add Polish translation:

```
SV-318 to podstawowy model z serii Spectravideo. Był wyposażony w klawiaturę typu chiclet, trudną w użyciu, obok której znajdowała się kombinacja kursora i joysticka. Jest to element w kształcie dysku z otworem w środku; po włożeniu czerwonego plastikowego "patyczka" w otwór jest to wbudowany joystick, po wyjęciu patyczka jest to strzałka kierunkowa do edytowania tekstu itp. Maszyna ta miała również tylko 16 KB pamięci RAM użytkownika (plus dodatkowe 16 KB pamięci RAM wideo), co ograniczało jej użyteczność, choć można ją było rozszerzyć za pomocą zewnętrznego modułu peryferyjnego.

Maszyna ta jest w zasadzie identyczna jak jej starszy brat SV-328, a jedyne różnice dotyczą klawiatury i ilości pamięci. ROM, możliwości rozbudowy, płyta główna i obudowa obu maszyn były identyczne.

Odniesienia do systemu operacyjnego Microsoft Extended BASIC nie należy mylić z MSX BASIC, chociaż niektórzy ówcześni marketingowcy twierdzili, że Microsoft Extended jest tym, co oznaczało MSX. SV-318 nie jest w pełni zgodny ze standardem MSX.
```

**Review by g-spawnPL — CHANGES_REQUESTED** [review]
Please resize the logo file so that its scale matches the templat
archimedes
atom
electron
user-images.githubusercontent.com/87877685/163869650-da0e19b1-1509-4a82-a4e4-ee0d3a040f68.png)

File: _inc/layouts/Spectravideo.xml (hunk 1 above)
Comment (Lendersmark) [reply _inc/layouts/Spectravideo.xml]: Thank you, translation added. If you notice some missing translations, please send them to me so I can update them.
(see "colorful-add-systems" branch in my repository: https://github.com/Lendersmark/ckau-book-enhanced/tree/colorful-add-systems )

**Comment by Lendersmark** [comment]
> Please resize the logo file so that its scale matches the template:

The logos you posted seem pretty identical to the ones in the theme. But I don't have these systems in my installation so I can't test them :/ could you please send me screenshots? (better in private message so we don't flood this tread - or open an issue in my repo https://github.com/Lendersmark/ckau-book-enhanced/issues )

**Comment by Lendersmark** [comment]
I'm going to close this pull request.
All changes have been merged to the master branch of my repo; another pull request has recently been made by @g-spawnPL which is basically identical to this one so let's keep that one and close this old one.

**Pull request closed without merging**